src/bosh-director/lib/bosh/director/api/controllers/events_controller.rb

```diff
@@ -6,6 +6,19 @@ module Api::Controllers
     class EventsController < BaseController
       EVENT_LIMIT = 200
 
+      get '/:id' do
+        content_type(:json)
+
+        event_id = params[:id]
+        event = Models::Event[event_id]
```

Comment:
I had a deeper look at it, and also don't understand the reason, why it works. So I've updated the code to feel more comfortable